BililiveRecorder.Core/Api/Http/HttpApiClient.cs

```diff
@@ -45,6 +55,8 @@ public HttpApiClient(GlobalConfig config)
             headers.Add("User-Agent", HttpHeaderUserAgent);
         }
 
+        long IDanmakuServerApiClient.GetUid() => this.Uid;
```

Comment:
```suggestion
        public long GetUid() => this.Uid;
```

移动到获取弹幕信息的 method 旁边